src/helpers/gho-launch/constants/GhoCCIPChains.sol

```diff
@@ -247,7 +297,31 @@ library GhoCCIPChains {
         aclManager: address(AaveV3InkWhitelabel.ACL_MANAGER),
         tokenAdminRegistry: CCIPChainTokenAdminRegistries.INK,
         owner: GovernanceV3Ink.EXECUTOR_LVL_1,
-        ccipRouter: CCIPChainRouters.INK
+        ccipRouter: CCIPChainRouters.INK,
+        linkToken: 0x71052BAe71C25C78E37fD12E5ff1101A71d9018F,
+        isVersion_1_6: false
+      });
+  }
+
+  /**
+   * @notice Returns the ChainInfo constant for Plasma
+   * @return The ChainInfo constant for Plasma
+   */
+  function PLASMA() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.PLASMA,
+        ghoToken: 0xb77E872A68C62CfC0dFb02C067Ecc3DA23B4bbf3,
+        ghoCCIPTokenPool: 0x360d8aa8F6b09B7BC57aF34db2Eb84dD87bf4d12,
+        ghoBucketSteward: 0x2Ce400703dAcc37b7edFA99D228b8E70a4d3831B,
+        ghoAaveCoreSteward: 0xA5Ba213867E175A182a5dd6A9193C6158738105A,
+        ghoCCIPSteward: 0x20fd5f3FCac8883a3A0A2bBcD658A2d2c6EFa6B6,
+        aclManager: address(AaveV3Plasma.ACL_MANAGER),
+        tokenAdminRegistry: CCIPChainTokenAdminRegistries.PLASMA,
+        owner: GovernanceV3Plasma.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.PLASMA,
+        linkToken: 0x76a443768A5e3B8d1AED0105FC250877841Deb40,
+        isVersion_1_6: true
```

Comment:
only plasma is migrated to 1.6 but not the others? the deployed plasma token pool is the same as other chains, 1.5.1, & the token pool contracts are supposed to backwards compatible with 1.6 infra